includes/admin/class-wcs-admin-system-status.php

```diff
@@ -118,10 +150,15 @@ private static function set_staging_mode( &$debug_data ) {
 	 * @param array $debug_data
 	 */
 	private static function set_live_site_url( &$debug_data ) {
+		// Use pre-determined SSR data if possible.
+		$site_url = isset( self::$report_data['live_url'] )
+			? self::$report_data['live_url']
+			: WCS_Staging::get_site_url_from_source( 'subscriptions_install' );
+
 		$debug_data['wcs_live_site_url'] = array(
 			'name'      => _x( 'Subscriptions Live URL', 'Live URL, Label on WooCommerce -> System Status page', 'woocommerce-subscriptions' ),
 			'label'     => 'Subscriptions Live URL',
-			'note'      => '<a href="' . esc_url( WCS_Staging::get_site_url_from_source( 'subscriptions_install' ) ) . '">' . esc_html( WCS_Staging::get_site_url_from_source( 'subscriptions_install' ) ) . '</a>',
+			'note'      => '<a href="' . esc_url( $site_url ) . '">' . esc_html( WCS_Staging::get_site_url_from_source( 'subscriptions_install' ) ) . '</a>',
```

Comment:
```suggestion
			'note'      => '<a href="' . esc_url( $site_url ) . '">' . esc_html( $site_url ) . '</a>',
```

Can we replace the second `WCS_Staging::get_site_url_from_source()` with `$site_url` as well?

Reply:
Uh, not sure how I missed that 🤦🏼 

Thanks for flagging, addressed in https://github.com/Automattic/woocommerce-subscriptions-core/pull/779/commits/d95a8a6ecf945f418dbec78a8e898a8257fa8862.